app/views/admin/users/show/profile/actions/_banish.html.erb

```diff
@@ -1,19 +1,12 @@
 <div id="banish-for-spam">
   <div class="flex flex-col gap-4">
-    <% if @user.comments.where("created_at < ?", 100.days.ago).empty? && @user.created_at < 100.days.ago %>
+    <% if (@user.comments.where("created_at < ?", 100.days.ago).empty? && @user.created_at < 100.days.ago) || current_user.super_admin? || current_user.support_admin? %>
```

Comment:
Refactor (non-blocking): Consider encapsulating this conditional into an instance variable like `can_banish_user`.

Reply:
Done! ✔️